src/Types.ts

```diff
@@ -36,6 +36,13 @@ export interface TabsterCoreProps {
      * This option allows to enable dummy inputs on Root.
      */
     rootDummyInputs?: boolean;
+    /**
+     * When Modalizer is active, we put aria-hidden to everything else to hide
+     * it from screen readers. This CSS selector allows to exclude some elements
+     * from this behaviour. For example, this could be used to exclude aria-live region
+     * with the application-wide status announcements.
+     */
+    modalizerAlwaysAccessible?: string;
```

Comment:
why is this an overall tabster property and not a specific modalizer property?

Reply:
because it is for the global application scenario. and having it here wouldn't require refactoring every modalizer used. and it basically is a workaround for the lack of browser functions (no JavaScript API to announce things via screen reader), so it will be good to be able to revoke it easily once the bright future. in short, the Modalizer is a legit application logic scenario. and aria-live containers in the body is just a workaround that is hoped to be temporarty.

Reply:
but users have to initialize the modalizer with `getModalizer` right? why not add the configuration there?